src/snowflake/cli/_app/snow_connector.py

```diff
@@ -46,6 +44,22 @@
 ENCRYPTED_PKCS8_PK_HEADER = b"-----BEGIN ENCRYPTED PRIVATE KEY-----"
 UNENCRYPTED_PKCS8_PK_HEADER = b"-----BEGIN PRIVATE KEY-----"
 
+# connection keys that can be set using SNOWFLAKE_* env vars
+SUPPORTED_ENV_OVERRIDES = [
+    "account",
+    "user",
+    "password",
+    "authenticator",
+    "private_key_file",
```

Comment:
Will it work for `private_key_path` alias?

Reply:
No, it won't. We lost that ability with this change: https://github.com/snowflakedb/snowflake-cli/pull/1506/files#diff-05f8159f2785119ba7d4251bb727ca88d0b874f88a650847ce5b86adf7ac13abR209

If we want to support both as aliases, I can add that back in here.

Reply:
@sfc-gh-astus my latest commit supports this alias, and I added a test to ensure that there's no errors when interacting with `_path` vs `_file` in env vars or in the config TOML. Using both is undefined behaviour (which one will "win" is up to iteration order), but allowed.